ViewDeck/IIViewDeckController.m

```diff
@@ -915,6 +915,38 @@ - (UIViewController *)childViewControllerForStatusBarStyle {
 
 #pragma mark - Appearance
 
+- (void)_forwardViewWillAppear:(BOOL)animated onViewController:(UIViewController *)viewController {
```

Comment:
I don't think Apple takes this suggestion very serious if you e.g. have a look at the class dump for `UIViewController`. However I consider using `_ii_` as a prefix next time.